### PR TITLE
Rework for PR#1449 switching cadc to use pyvo

### PR DESCRIPTION
**Pull request by andamian**
PR#1449

### CODE REVIEW

**Comment by andamian** [comment]
@keflavich @bsipocz - The rework after the code review. A bit late - sorry.

**Comment by andamian** [comment]
@keflavich @bsipocz - PR https://github.com/astropy/astroquery/pull/1486 is also ready for your review. If you wise we could merge the changes in this PR into that one since they are related and drop this one. Please let me know. Thanks

File: astroquery/cadc/tests/test_cadctap.py [hunk 1]
```diff
@@ -60,7 +62,7 @@ def test_get_tables():
 @pytest.mark.skipif(not pyvo_OK, reason='not pyvo_OK')
 def test_get_table():
     table_set = PropertyMock()
-    tables_result = [Mock(), Mock(), Mock()]
+    tables_result = [Mock() for _ in range(3)]
```
Comment (keflavich) [inline astroquery/cadc/tests/test_cadctap.py]: awww, I liked the http://dontknockmysmock.com sound.  But, this is fine.

**Comment by bsipocz** [comment]
Thanks @andamian!

**Pull request merged**